public/app/features/alerting/unified/RuleViewer.tsx

```diff
@@ -17,10 +20,51 @@ type RuleViewerProps = GrafanaRouteComponentProps<{
 
 const newAlertDetailView = Boolean(config.featureToggles.alertingDetailsViewV2) === true;
 
-const RuleViewer = (props: RuleViewerProps): JSX.Element => (
-  <AlertingPageWrapper>
-    {newAlertDetailView ? <DetailViewV2 {...props} /> : <DetailViewV1 {...props} />}
-  </AlertingPageWrapper>
-);
+const RuleViewer = (props: RuleViewerProps): JSX.Element => {
+  return newAlertDetailView ? <RuleViewerV2Wrapper {...props} /> : <RuleViewerV1Wrapper {...props} />;
+};
+
+export const defaultPageNav: NavModelItem = {
+  id: 'alert-rule-view',
+  text: '',
+};
+
+const RuleViewerV1Wrapper = (props: RuleViewerProps) => <DetailViewV1 {...props} />;
```

Comment:
with this change we are skipping wrapping `DetailViewV1` with `AlertingPageWrapper`. Is this on purpose?

Reply:
The way the old alert detail view is wrapped is quite odd – if you look closely it seems to have the page nested (see screenshot). 

I'll append a change to have just a single page wrapped around it to get around it.

<img width="944" alt="image" src="https://github.com/grafana/grafana/assets/868844/e4b0165a-b4ec-4a14-b86a-ee63035b209d">